Framework/Algorithms/test/UnwrapMonitorTest.h

```diff
@@ -0,0 +1,212 @@
+#ifndef MANTID_ALGORITHMS_UNWRAPMONITORTEST_H_
+#define MANTID_ALGORITHMS_UNWRAPMONITORTEST_H_
+
+#include <cxxtest/TestSuite.h>
+#include "MantidAPI/AnalysisDataService.h"
+#include "MantidAPI/Axis.h"
+#include "MantidDataObjects/Workspace2D.h"
+#include "MantidTestHelpers/WorkspaceCreationHelper.h"
+
+#include <string>
+
+#include "MantidAlgorithms/UnwrapMonitor.h"
+
+using namespace Mantid::Algorithms;
+using namespace Mantid::API;
+using namespace Mantid::DataObjects;
+using namespace Mantid::Kernel;
+
+class UnwrapMonitorTest : public CxxTest::TestSuite {
+private:
+  void makeFakeWorkspace(std::string wsName) {
```

Comment:
This could return a `const MatrixWorkspace_sptr` and `wsName` can then be dropped.